sources/packages/backend/libs/integrations/src/esdc-integration/msfaa-integration/msfaa.integration.service.ts

```diff
@@ -143,8 +144,10 @@ export class MSFAAIntegrationService extends SFTPIntegrationBase<MSFAASFTPRespon
     const client = await this.getClient();
     try {
       const filePath = `${this.esdcConfig.ftpResponseFolder}/${fileName}`;
-      // Read all the file content and create a buffer.
-      const fileContent = await client.get(filePath);
+      // Read all the file content and create a buffer with 'ascii' encoding.
+      const fileContent = await client.get(filePath, undefined, {
```

Comment:
@andrewsignori-aot @andrepestana-aot @sh16011993 @lewischen-aot 

Not a part of the PR, we need to change this code to use sftp-inregration-base.ts unless we have a reason to keep this code here.

Reply:
@dheepak-aot From what I understand, the base class `sftp-integration-base.ts` does not carry the implementation for this method `downloadResponseFile`. All the subclasses are having their own specific implementations for this method. 

Not sure if you meant something else.

Reply:
Thanks @dheepak-aot for clarification on the call. This method can make use of the `downloadResponseFileLines` from the base class `sftp-integration-base.ts`. I agree!

Reply:
@dheepak-aot is is probably a leftover from before the base class. We can have a tech debt created for it.

Reply:
I can create a tech debt ticket.